server/events/vcs/common/common_test.go

```diff
@@ -41,8 +41,8 @@ func TestSplitComment_TwoComments(t *testing.T) {
 	expFirstComment := comment[:expCommentLen]
 	expSecondComment := comment[expCommentLen:]
 	Equals(t, 2, len(split))
-	Equals(t, expFirstComment+sepEnd, split[0])
-	Equals(t, sepStart+expSecondComment, split[1])
+	Equals(t, sepStart+expSecondComment, split[0])
+	Equals(t, expFirstComment+sepEnd, split[1])
```

Comment:
Have you tested this end to end?

Your pr body just says "yes". Please expand on that.